test-app/src/app.ts

```diff
@@ -53,3 +55,17 @@ peopleDb.deletePerson.addResolver(api, {
   typeName: "Mutation",
   fieldName: "deletePerson",
 });
+
+type MyEvent = EventBusRuleInput<{
```

Comment:
interface instead of type? Which way are we going to encourage?

Reply:
interfaces are not supported in this form. 

> An interface can only extend an identifier/qualified-name with optional type arguments. ts(2499)

The options are

```ts
interface MyEventDetails {
  value: string;
}

interface MyEvent extends EventBusRuleInput<MyEventDetails>{};
```

or 

```ts
type MyEvent = EventBusRuleInput<{
   value: string;
}>;
```

I went with the less verbose way, but fine with the former.